data-structures/linkedList/src/main/java/linkedList/LinkedList.java

```diff
@@ -0,0 +1,113 @@
+package linkedList;
+
+public class LinkedList {
+/****************
+ * Instance Variables
+ * */
+    private int size;
+    private Node head;
+
+
+/****************
+ * Constructors
+ * */
+    public LinkedList() {
+        this.head = null;
+        this.size = 0;
+    }
+
+
+/****************
+ * Getters & Setters
+ * */
+    public int getSize() { return size; }
+    public Node getHead() { return head; }
+
+    private void setSize() { this.size++; }
+    private void setHead(Node h) { this.head = h; }
+
+
+
+/****************
+ * Insert
+ *
+ * This method takes a value, creates a new Node, and sets that Node at the head of the linked list.
+ * */
+    public void insert(int v) {
+        try {
```

Comment:
I'm not 100% sure the try / catch is necessary here. The code won't compile if someone tries to pass in anything other than an int